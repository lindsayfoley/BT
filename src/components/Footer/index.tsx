import { FunctionComponent } from 'react';

const Footer: FunctionComponent = () => (
  <footer>
    <small>BT Tech Test 2021</small>
  </footer>
);

export default Footer;
