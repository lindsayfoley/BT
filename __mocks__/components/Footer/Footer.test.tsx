import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import Footer from '../../../src/components/Footer';

const setup = () => shallow(<Footer />);

describe('<Footer />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = setup();
  });
  
  it('renders without crashing', () => {
    expect(wrapper.isEmptyRender()).toBe(false);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render small print in footer', () => {
    expect(wrapper.find('small')).toHaveLength(1);
  });
});
