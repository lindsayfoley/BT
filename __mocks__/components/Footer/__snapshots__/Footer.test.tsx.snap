// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders without crashing 1`] = `
<footer>
  <small>
    BT Tech Test 2021
  </small>
</footer>
`;
